### PR TITLE
[installer]: fix jaeger operator misconfiguration

### LINKED ISSUE #6759: [installer]: fix jaeger-operator misconfigs
There's misconfiguration in the Jaeger operator:

- [ ] the `values.yaml` has the `docker-registry` config [1](https://gitpod.slack.com/archives/C02F19UUW6S/p1637153356276400?thread_ts=1637153013.276100&cid=C02F19UUW6S)
- [ ] the `helm.go` values don't begin with `jaeger-operator`, so never applied [2](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/jaeger-operator/helm.go#L21)

The fact that it currently works with Jaeger doesn't make much sense, but need to apply the correct values

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

Fixed copy/pasta errors in the Jaeger operator

## Related Issue(s)

Fixes #6759

## How to test

Update the config:

```yaml
observability:
  logLevel: info
  tracing:
    endpoint: http://jaeger-collector:14268/api/traces
```

Deploy [Jaeger](https://github.com/gitpod-io/gitpod/blob/ee92ed0de957b14035a9dcd460b50f4f57c32f40/.werft/jaeger.yaml#L35), changing the `nodeAffinity` key to `gitpod.io/workload_meta`

Create a port forwarder to the jaeger deployment `kubectl port-forward jaeger-5945545574-g5jgc 3000:16686`.

Go to [localhost:3000](HTTP://localhost:3000) and check that there are traces flowing in

## Release Notes

```release-note
[installer]: fix jaeger operator misconfiguration
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mrsimonemms** after the PR has been reviewed.

Associated issue: *#6759*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: installer/pkg/components/jaeger-operator/helm.go [hunk 1]
```diff
@@ -18,9 +18,8 @@ var Helm = common.CompositeHelmFunc(
 			Enabled: pointer.BoolDeref(cfg.Config.Jaeger.InCluster, false),
 			Values: &values.Options{
 				Values: []string{
-					"installCRDs=true",
-					"crd.install=false",
-					"rbac.clusterRole=true",
+					helm.KeyValue("jaeger-operator.installCRDs", "true"),
```
Comment (kylos101) [inline installer/pkg/components/jaeger-operator/helm.go:21]: I _think_ `installCRDs` should be `crd.install` per the operator's [config table](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger-operator#configuration). Unless of course I've missing something? Lemme know whatcha think.

I am guessing `installCRDs` is related to prior `cert-manager` [stuffs](https://github.com/gitpod-io/gitpod/blob/main/chart/values.yaml#L758).

**Review by kylos101 — COMMENTED** [review]
Hey @MrSimonEmms just one question, lemme know whatcha think?

File: installer/pkg/components/jaeger-operator/helm.go (hunk 1 above)
Comment (MrSimonEmms) [reply installer/pkg/components/jaeger-operator/helm.go:21]: Yeah, I was rushing at the end of the day. Thanks for saving me from myself

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6760#issuecomment-972897649" title="Approved">kylos101</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mrsimonemms** after the PR has been reviewed.

Associated issue: *#6759*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6760#issuecomment-972897649" title="Approved">kylos101</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6760#issuecomment-972906428" title="Approved">MrSimonEmms</a>*

Associated issue: *#6759*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [MrSimonEmms]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: d73006177635f65b7d38af9fa7087eafc0033656</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6760#issuecomment-972923938" title="LGTM">kylos101</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6760#issuecomment-972906428" title="Approved">MrSimonEmms</a>*

Associated issue: *#6759*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [MrSimonEmms]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**